### PR TITLE
Update readme about usage

### PR DESCRIPTION
**Pull request by Jintin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ There are three way to install.
 
 **After installation, you should open the `Swimat.app` once to make the functionality works.**
 
-In the menu open **[Editor] -> [Swimat] -> [Format]** or map it to a hot-key you like in Xcode's Key Binding preferences.
+In the Xcode menu click **[Editor] -> [Swimat] -> [Format]** then the current active file will reformat.
+You can also create a hot-key in **[Xcode] -> [Preferences..] -> [Key Bindings]**, if you don't have any prefernce you can set as [⌘ + ⇧ + L].
```
Comment (saagarjha) [inline README.md]: Does GitHub markdown render `<kbd></kbd>`?

Comment (Jintin) [reply README.md]: I think so and good point.

Comment (Jintin) [reply README.md]: Screen Shot 2019-09-06 at 11 52 57 PM
Though it's not as beauty as I expect.

Comment (saagarjha) [reply README.md]: Yeah, it's not the nicest looking CSS :(

**Pull request merged**